tests/test_nodes.py

```diff
@@ -51,6 +52,7 @@
     "call(something=something)",
     # Strings.
     "f'a {round(key, 2)} {z}'",
+    *(["t'a {round(key, 2)} {z}'"] if sys.version_info >= (3, 14) else []),
```

Comment:
could also try something like
```suggestion
    pytest.param("t'a {round(key, 2)} {z}'", marks=pytest.mark.skipif(sys.version_info < (3, 14))),
```